dev/opentitan.nix

```diff
@@ -0,0 +1,68 @@
+{
+  pkgs,
+  ncurses5-fhs,
+  python_ot,
+  verilator_ot,
+  edaTools ? [],
+  ...
+}: let
+  edaExtraDeps = with pkgs; [elfutils openssl];
+in
+  (pkgs.buildFHSEnv {
+    name = "opentitan";
+    targetPkgs = _:
+      with pkgs;
+        [
```

Comment:
Could you pull out the wrapped/overriden packages into a let binding?

Reply:
Done